### DISCUSSION TITLE
Enable edit rules

### DISCUSSION THREAD

**Discussion by aruznieto** [key-networks/ztncui#129] [Ideas]
It is possible to add rules editor?

**Comment by key-networks** [comment]
A start was made here https://github.com/key-networks/ztncui/pull/76 but was never completed.  Would you be willing to test what has been done thus far and let us know if it is fit for consumption?

**Reply by aruznieto** [reply]
Yes, if you can say me how can I test it, I would be pleased to do so.

**Reply by key-networks** [reply]
I am trying to build the RPM and DEB files for you, but am having trouble with running the VM for the build host.  This will take longer than anticipated.

**Reply by aruznieto** [reply]
Why don't you try to run it on GitHub Actions?

**Reply by key-networks** [reply]
@aruznieto, please try the following, which are hot off the press and I have not tried them at all.

RPM:
`sudo dnf --enablerepo=ztncui-testing upgrade ztncui`

DEB:
`https://key-networks.s3.us-west-1.amazonaws.com/deb/ztncui/1/x86_64/ztncui_0.0.0_amd64.deb`

**Reply by aruznieto** [reply]
Ok, I´m going to try, what is the default password?

**Reply by aruznieto** [reply]
It does not work.

1.- In the member name appears that

<img width="305" alt="image" src="https://github.com/key-networks/ztncui/assets/40019177/0f71ac82-8d98-4392-b1c8-ba910cb4983c">

2.- When you add some tag, and you click on Save, it does not save on the configuration. In addition, you can't add a tag to users

**Reply by aruznieto** [reply]
I´m working on it, but when I am trying to edit "rulesSource" it does not change

**Reply by lideming** [reply]
Hopefully I will make it work in weeks.

**Reply by aruznieto** [reply]
I´m doing, something if you want we can talk throw telegram/discord.

**Reply by lideming** [reply]
t(dot)me/realYuuza
